CMakeLists.txt

```diff
@@ -680,6 +681,11 @@ elseif(ANDROID)
       endif()
     endif()
 
+    if(LOVR_USE_OCULUS_AUDIO)
+      get_target_property(OCULUS_AUDIO_LIB ${LOVR_OCULUS_AUDIO} IMPORTED_LOCATION)
+      file(COPY ${OCULUS_AUDIO_LIB} DESTINATION raw/lib/${ANDROID_ABI})
+    endif()
+
```

Comment:
I don't like this, this is duplicative with the VRAPI case. The best way to do this would be to create a function like we have with move_dll, and use it with both the VRAPI/PICO lines and also this.

Reply:
Seems okay, there's like 4 `move_lib` functions that should try to get merged into one eventually.